Mathlib/Tactic/Continuity.lean

```diff
@@ -13,7 +13,7 @@ import Mathlib.Algebra.Group.Defs
 We define the `continuity` tactic using `aesop`. -/
 
 attribute [aesop (rule_sets [Continuous]) unfold norm] Function.comp
-attribute [aesop (rule_sets [Continuous]) unfold norm] npowRec
```

Comment:
Maybe worth a TODO comment (here and in the other file): `npowRec` is an implementation detail, and certainly not something aesop should know about.